lib/main.dart

```diff
@@ -26,109 +25,112 @@ class MeasurementScreen extends StatefulWidget {
 }
 
 class _MeasurementScreenState extends State<MeasurementScreen> {
-  late Stream<Vector3> _stream;
   Vector3? _firstValue;
   Vector3? _lastValue;
-  double distance = 0.00;
-  late StreamSubscription<Vector3> _subscription;
-  bool _isMeasuring = false;
+  late StreamSubscription<Vector3>? _subscription;
+  bool get isMeasuring => _subscription != null;
+  double _distance = 0.0;
+  String _replacementResult = '';
 
   @override
   void initState() {
     super.initState();
-    _stream = ArkitTracer.trace();
-     _subscription = _stream.listen(
+    _subscription = null;
+  }
```

Comment:
`initState` can be removed together with the `late` keyword on line 30: `late StreamSubscription<Vector3>? _subscription;`